src/domain/players/AIPlayer.js

```diff
@@ -0,0 +1,15 @@
+const AIPlayer = () => {
+    const player = {
+        onQuestionAsked: ask => {
+            const question = ask();
+            const randomId = Math.floor(Math.random() * 4);
```

Comment:
```suggestion
            const randomId = Math.floor(Math.random() * 4);
```
zerknij na funkcję dodaną przez @Olaf-Koziara (getRandomInt)